use-cases.md

```diff
@@ -206,16 +207,16 @@ def traced_request(request, operation, http_client):
   * Because the HTTP client may throw an exception even before returning a Future, we use a try/catch block to finish the Span in all circumstances, to ensure it is reported and avoid leaking resources.
 
 
-### Using Distributed Context Propagation
+### Using Baggage / Distributed Context Propagation
 
 The client and server examples above propagated the Span/Trace over the wire, including any Baggage. The client may use the Baggage to pass additional data to the server and any other downstream server it might call.
 
 {% highlight python %}
 # client side
-span.set_baggage_item('auth-token', '.....')
+span.context.set_baggage_item('auth-token', '.....')
```

Comment:
@adriancole we should talk about it IRL in July (or before over VC, whatever)... an interesting evolution to this point, but I think exposing an explicit context object solves lots of corner-case problems and points of confusion with OpenTracing, esp around propagation, more exotic span-to-span references, and use-after-finish timing issues with Spans.